### PR TITLE
DMG desc

### PR DESCRIPTION
**Pull request by FerretDave**
improved/added some item descriptions.

### CODE REVIEW

File: data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst [hunk 1]
```diff
@@ -596,7 +596,7 @@ Ring of Force Shield															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST
...
 Ring of Friend Shield (1 of 2)													OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:25000	WT:0								SOURCEPAGE:p.232																																																				SPROP:These curious rings always come in pairs. A friend shield ring without its mate is useless. Either wearer of one of a pair of the rings can, at any time, command his or her ring to cast a shield other spell with the wearer of the mated ring as the recipient. This effect has no range limitation. Moderate abjuration; CL 10th
-Ring of Invisibility															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:20000	WT:0								SOURCEPAGE:p.232												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=3|Invisibility																															SPROP:By activating this simple silver ring, the wearer can benefit from invisibility, as the spell (PH 245). Faint illusion; CL 3rd
+Ring of Invisibility															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:20000	WT:0								SOURCEPAGE:p.232												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=3|Invisibility																															SPROP:By activating this simple silver ring, the wearer can benefit from invisibility, as the spell (PH 245)
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst:599]: Why is the faint illusion; CL 3rd being removed?

**Review by BahamutDragon — CHANGES_REQUESTED** [review]
Deviation from rest of file noted.

File: data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst (hunk 1 above)
Comment (FerretDave) [reply data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst]: that was in error, corrected.

File: data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst [hunk 2]
```diff
@@ -1747,12 +1747,13 @@ Figurine of Wondrous Power (Ivory Goats)											OUTPUTNAME:Figurine of Wondro
...
+Figurine of Wondrous Power (Onyx Dog).MOD	SPROP:Riding Dog:Hit Dice: 2d8+4 (13 hp),Initiative: +2, Speed: 40 ft, Armor Class: 16 (+2 Dex, +4 natural), touch 12, flat-footed 14, Base Attack/Grapple: +1/+3, Attack: Bite +3 melee (1d6+3), Full Attack: Bite +3 melee (1d6+3), Space/Reach: 5 ft./5 ft. Special Qualities: Low-light vision, scent, Saves: Fort +5, Ref +5, Will +1,  Abilities: Str 15, Dex 15, Con 15, Int 2, Wis 12, Cha 6,  Skills: Jump +8, Listen +5, Spot +5, Swim +3, Survival +1, Feats: Alertness, Track. Carrying Capacity: light 100, medium load 101–200, ; and a heavy load 201–300 pounds. A riding dog can drag 1,500 pounds.	PRERULE:1,DisplayFullEquip
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst:1750]: I don't think this is going to have the desired outcome. The PRERULE is not attached to anything.

Comment (FerretDave) [reply data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst:1750]: without the pipe it's disabling the entire MOD line, thus not adding the SPROP to the default SPROP -

**Review by BahamutDragon — APPROVED** [review]

**Pull request merged**